app/models/recipe.rb

```diff
@@ -0,0 +1,5 @@
+class Recipe < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_foods, dependent: :destroy
+  has_many :foods, through: :recipe_foods
+end
```

Comment:
- [ ] Please validate the input fields for your forms.

Reply:
Validations are added to required places!